device_samples/getting_started/tslint.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
We want to move everything over to ESLint at some point, so maybe we should use ESLint here. But also it might be better to keep everything consistent and move everything over to ESLint in one go.

Reply:
I will leave this to you if you want to move to ESList later